timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -225,6 +230,7 @@ private AsyncTimelockService createAsyncTimelockService(
     private AsyncTimelockService createRawAsyncTimelockService(
             String client,
             Supplier<ManagedTimestampService> timestampServiceSupplier) {
+        log.info("Creating async timelock service.");
```

Comment:
one small caveat here, this method is called by `AwaitingLeadershipProxy`so this will not get logged until we are the leader - might want to move this into `createTimeLockServicesWithAsync`